mobile/src/components/uam-configure-group/components/users-table/users-table.tsx

```diff
@@ -21,14 +21,21 @@ type Props = {
     page: number;
     setPage: (page: number) => void;
   };
+  checkedIds?: number[] | [];
```

Comment:
```suggestion
  checkedIds?: number[];
```

Reply:
I was already trying to change this, but my network was bad) TY!